frontend/src/hooks/useDjangoSession.ts

```diff
@@ -24,34 +25,60 @@ export const useDjangoSession = () => {
   const [isSyncing, setIsSyncing] = useState(false)
 
   useEffect(() => {
-    // If the session is unauthenticated, remove the sync status flag
-    // so next login re‑runs sync
     if (status === 'unauthenticated') {
       sessionStorage.removeItem(SYNC_STATUS_KEY)
       return
     }
 
     const shouldSync =
-      status === 'authenticated' && //logged in via jwt
-      (session as ExtendedSession)?.accessToken && // we have GH Token
-      !sessionStorage.getItem(SYNC_STATUS_KEY) //not yet synced
+      status === 'authenticated' &&
+      (session as ExtendedSession)?.accessToken &&
+      !sessionStorage.getItem(SYNC_STATUS_KEY)
 
     if (shouldSync) {
       setIsSyncing(true)
 
-      // sends  Github token to Django.
-      // On Success, Django writes session cookie, browser stores it
-
+      // The backend response contains a backend session cookie if successful.
+      // The cookie name is set in SESSION_COOKIE_NAME of backend/settings/base.py.
       syncSession({
         variables: {
           accessToken: (session as ExtendedSession).accessToken,
         },
       })
-        .then(() => {
-          sessionStorage.setItem(SYNC_STATUS_KEY, 'true')
+        .then((response) => {
+          const githubAuth = response?.data?.githubAuth
+          if (githubAuth?.ok) {
+            sessionStorage.setItem(SYNC_STATUS_KEY, 'true')
+            addToast({
+              color: 'success',
+              description: githubAuth?.message,
+              shouldShowTimeoutProgress: true,
+              timeout: 3000,
+              title: 'Authentication Successful',
+              variant: 'solid',
```

Comment:
What if we change this to `bordered`? With current `solid` color the `showTimeoutProgress` is not really visible.

```suggestion
              variant: 'bordered',
```
<img width="397" height="124" alt="Screenshot 2025-07-15 at 5 24 25 PM" src="https://github.com/user-attachments/assets/6253ecf7-66cf-44a4-b3ab-e1a96d05cd9e" />
<img width="405" height="133" alt="Screenshot 2025-07-15 at 5 16 19 PM" src="https://github.com/user-attachments/assets/1a822f38-a66a-4004-83ea-d265548ef82e" />